### PR TITLE
Dtspb 902 change page 2 sols appy

### PR DESCRIPTION
**Pull request by meganaslattery**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-902

### Change description ###

Grant of Probate -Changes to Step 2 'Apply for Probate' page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json [hunk 1]
```diff
@@ -64,16 +64,16 @@
...
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorPhoneNumber", "Label": "Contact phone number", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAppReference", "Label": "Your reference for this application", "FieldType": "Text", "SecurityClassification": "Public", "Max": 100},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsExec", "Label": "Is the qualified legal professional named in the will as an executor?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorWillSignSOT", "Label": "Are you the qualified legal professional who will sign the statement of truth?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsMainApplicant", "Label": "Is the solicitor the main applicant?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
```
Comment (vinceholt) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:74]: We should remove this field as well shouldn't we?

**Review by vinceholt — COMMENTED** [review]
Wwe also need to remove solsSolicitorIsMainApplicant field I believe, and also the fields need completely removing from json test files, test code and fieldshowconditions

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json (hunk 1 above)
Comment (meganaslattery) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:74]: Shall we do this in the general clean up ticket? I think it's still being used in the CW journey

File: ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json [hunk 2]
```diff
@@ -1,10 +1,7 @@
...
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "Renunciation", "ListElement": "They have renounced"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerOfAttorney", "ListElement": "They have appointed or wish to appoint another person to act as their attorney"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "MentallyIncapable", "ListElement": "They lack capacity to act as executor"},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerReserved", "ListElement": "They are holding power reserved", "DisplayOrder": 1},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "Renunciation", "ListElement": "They have renounced", "DisplayOrder": 2},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerOfAttorney", "ListElement": "They have appointed or wish to appoint another person to act as their attorney", "DisplayOrder": 3},
```
Comment (KleoG) [inline ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json]: I remember something about these fixed lists and there was uncertainty about what should be involved in it

**Review by KleoG — APPROVED** [review]

File: ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json (hunk 2 above)
Comment (meganaslattery) [reply ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json]: Oh I forgot! Thanks Kleo!

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json (hunk 1 above)
Comment (vinceholt) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json:74]: ok! CW journey needs to change too so will remove from there in the clean-up

**Review by vinceholt — APPROVED** [review]

**Pull request merged**